left/right_wheel_jointのVelocityJointInterface指定方法で警告が出る

<!-- 品質向上にご協力ありがとうございます -->
<!-- テンプレートを参考に文章を作成してください -->

# 不具合の概要
<!-- 不具合の概要をここに書きます -->

* `hardware_interface`の`VelocityJointInterface`の指定方法が古いため警告が出る
* `velocity_controllers`にPIDゲインが設定されていないのでエラーが出る（この設定はraspimouse_control内の設定）

# 実行環境

Ubuntu 18.04 + ROS Melodic

https://github.com/rt-net/raspimouse_sim の `7f001794bd6e39454467840cd88bd54495c4a789` から `raspimouse_description`を削除したもの + https://github.com/rt-net/raspimouse_description の `0bc902d0bdffceb15da2691b9aa1242c9a80ec30`


# 再現方法
<!-- 不具合の再現方法をここに書きます -->

1. 上記環境で`roslaunch raspimouse_gazebo open_emptyworld.launch`

# 期待する動作
<!-- 正常時にどんな動作をしてほしいのかをここに書きます -->

ワーニングやエラーが出ない

# ログ・画像

```
[ WARN] [1589795683.995359374, 0.267000000]: Deprecated syntax, please prepend 'hardware_interface/' to 'VelocityJointInterface' within the <hardwareInterface> tag in joint 'right_wheel_joint'.
[ERROR] [1589795683.996138034, 0.267000000]: No p gain specified for pid.  Namespace: /raspimouse_on_gazebo/gazebo_ros_control/pid_gains/right_wheel_joint
[ WARN] [1589795683.996180612, 0.267000000]: Deprecated syntax, please prepend 'hardware_interface/' to 'VelocityJointInterface' within the <hardwareInterface> tag in joint 'left_wheel_joint'.
[ERROR] [1589795683.996636194, 0.267000000]: No p gain specified for pid.  Namespace: /raspimouse_on_gazebo/gazebo_ros_control/pid_gains/left_wheel_joint
```

[![Image from Gyazo](https://i.gyazo.com/40802a38243ea06d6f3a3c96bd13b3ad.png)](https://gyazo.com/40802a38243ea06d6f3a3c96bd13b3ad)

--- a/urdf/raspimouse.urdf.xacro
+++ b/urdf/raspimouse.urdf.xacro
@@ -50,8 +50,8 @@
   <!-- <xacro:wheel_trans prefix="left" interface="PositionJointInterface"/>  -->
   <!-- VelocityJointInterface: requires position, velocity -->
   <!-- for DiffDriveController -->
-  <xacro:wheel_trans prefix="right" interface="VelocityJointInterface"/>
-  <xacro:wheel_trans prefix="left" interface="VelocityJointInterface"/>
+  <xacro:wheel_trans prefix="right" interface="hardware_interface/VelocityJointInterface"/>
+  <xacro:wheel_trans prefix="left" interface="hardware_interface/VelocityJointInterface"/>
   <!-- EffortJointInterface: requires position, velocity, effort -->
   <!-- <xacro:wheel_trans prefix="right" interface="EffortJointInterface"/> -->
   <!-- <xacro:wheel_trans prefix="left" interface="EffortJointInterface"/>  -->

--- a/urdf/raspimouse_urg.urdf.xacro
+++ b/urdf/raspimouse_urg.urdf.xacro
@@ -53,8 +53,8 @@
   <!-- ===============  Transmission =============== -->
   <!-- <xacro:wheel_trans prefix="right" interface="EffortJointInterface"/> -->
   <!-- <xacro:wheel_trans prefix="left" interface="EffortJointInterface"/> -->
-  <xacro:wheel_trans prefix="right" interface="VelocityJointInterface"/>
-  <xacro:wheel_trans prefix="left" interface="VelocityJointInterface"/>
+  <xacro:wheel_trans prefix="right" interface="hardware_interface/VelocityJointInterface"/>
+  <xacro:wheel_trans prefix="left" interface="hardware_interface/VelocityJointInterface"/>
 
   <!-- =============== Gazebo =============== -->
   <gazebo>

